box_rules/box_item_shared_externally.yml

```diff
@@ -0,0 +1,52 @@
+AnalysisType: rule
+Filename: box_item_shared_externally.py
+RuleID: Box.Item.Shared.Externally
+DisplayName: Box item shared externally
+Enabled: false
+LogTypes:
+  - Box.Event
+Tags:
+  - Box
+Severity: Medium
+Description: >
+  A user has shared an item and it is accessible to anyone with the share link (internal or external to the company). 
```

Comment:
I'd add a note to the description that this relies on the box sdk being installed.